libs/auth-api-lib/src/lib/delegations/models/delegation-type.model.ts

```diff
@@ -62,6 +64,9 @@ export class DelegationTypeModel extends Model<
   @BelongsToMany(() => Client, () => ClientDelegationType)
   clients!: Client[]
 
+  @BelongsToMany(() => ApiScope, () => ApiScopeDelegationType)
+  apiScopes!: ApiScope[]
```

Comment:
Here you want to make this field creation optional so we will not have to add this field to the creation model.
```suggestion
  apiScopes!: CreationOptional<ApiScope[]>
```